### DISCUSSION TITLE
Suggestion: Permissions on Issues

### DISCUSSION THREAD

**Discussion by gilgongo** [community/community#3397] [Discussions]
Now that Discussions is live, we have a great opportunity to make our Issues an actual list of defined bugs and feature enhancements. Right now, because anyone can post a issue, it's a mix of issues and open-ended discussions about potential issues. This makes it hard to manage because a lot of the time the name and description of the issue isn't what has been discussed below it.

It would be great if we could restrict the posting of Issues to only team members, while everyone else can open a Discussion in which a decision can be made about something. At that point a team member can then spec it up properly and add it to Issues, where we can easily apply meaningful tags, milestones, epics, etc.

(Someone's going to tell me that you can in fact restrict Issues posting to teams, aren't they?)

**Comment by ghost** [comment]
Hi, my suggestion is to add a code to move and leave a script for the sticker characters

Example:

🐈

name(🐈):cat
cat:Move to the right

**Reply by gilgongo** [reply]
Love that! Random and totally irrelevant to my suggestion though is :-)
